### DISCUSSION TITLE
Does anybody know how to loop a stream using gstreamer?

### DISCUSSION THREAD

**Discussion by antoniomtz** [bluenviron/mediamtx#476] [Questions]
Hello,

I would like to keep looping the video stream using gstreamer. Does anybody know if there is a plugin to do that?

**Comment by aler9** [comment]
The simplest way i know consists in:

1. remux the video in a MPEG-TS container (i.e. it must end with .ts)
2. use `multifilesrc`

```
gst-launch-1.0 multifilesrc location=myfile.ts loop=true ! decodebin ! rtspclientsink name=s location=rtsp://localhost:8554/mystream
```

[a comment by an automated account is not shown]
